dev-tools/command-line-advanced.md

```diff
@@ -271,6 +271,40 @@ This lesson helps you create an executable script. It will read information from
 
 Greg's Wiki is full of common mistakes (e.g. [why you shouldn't parse ls](http://mywiki.wooledge.org/ParsingLs)).
 
+### Installing Homebrew
+
+Now you know how to move using your command line, let's get deep into [Homebrew](https://brew.sh/). It's a package manager for masOS or Linux which provides a simple way to install programs or tools, similar to an app store for CLI.
+
+Before getting started, check that the following requirements are fullfilled:
+|macOS   |Linux   |
+|------------------|-------------|
+|**64-bit Intel** CPU | **64-bit x86_64** CPU |
+| Compatible shell (`.bash` or **zsh**)|  **GCC** 4.7.0 or newer  |
+|**macOS** 10.13 or newer|**Linux** 2.6.32 or newer|
+|Command Line Tools for [Xcode](https://apps.apple.com/us/app/xcode/id497799835)|**Glibc** 2.13 or newer|
+
+Then, installation will take three steps:
+1. Open a macOS Terminal or Linux shell prompt.
+2. Run the install script :
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+3. Verify your installation checking its version: `brew --version`. If no errors appear, everything went perfectly.
```

Comment:
Please add a step to run `brew doctor` as the types of packages we'll install will require building from source, so better they do this now rather than later.